docs/evaluation/simulate.rst

```diff
@@ -136,10 +221,14 @@ complex models which may include such components as the instrumental
 background (which should not be folded through the ARF) or arbitrary
 other components::
 
-  >>> fake_pha(data, model=inst_bkg + my_arf(my_rmf(srcmodel)), is_source=False)  # doctest: +SKIP
+   >>> fake_pha(data, model=inst_bkg + my_arf(my_rmf(srcmodel)), is_source=False)  # doctest: +SKIP
```

Comment:
`is_source` is not longer used, so I don't thing we should set it at all here, neither to True or False. The background is part of the model.

Reply:
And the first half of the paragraph above can probably be deleted.

Reply:
So, this change is my attempt to re-work documentation that I may have started but I'm pretty sure you improved at some point. So remembering what is versus what we want has been confusing for me here!